content/collections/data/en/amplitude-data-settings.md

```diff
@@ -89,4 +93,24 @@ You can tell Amplitude how to handle these situations by configuring your schema
 
 For any unplanned events, event properties, event property types, user properties, or user property types, you can tell Amplitude Data to either mark them as **unexpected**, or to **reject** them outright. Amplitude Data ingests any events or properties marked as unexpected and send a notification to everyone subscribed to this schema. If you choose to reject unexpected data, however, Amplitude Data doesn't ingest or store the rejected data. Subscribers still receive a notification.
 
-Click *Save* to implement any changes you make to your schema settings.
\ No newline at end of file
+Click _Save_ to implement any changes you make to your schema settings.
+
+## The Permissions Tab
+
+Your company may want to limit the number of people who have access to perform Data Management functions. For example, you may want most of your users to have the Member role so they can create dashboards and charts but prefer that custom event creation and other data management features be limited to managers or administrators.
+
+In the permissions tab, you can add additional restrictions for selected roles, limiting data management access while keeping other permissions for those roles.
+
+Under Restrict access in Amplitude Data, you can specify from the following options to restrict permissions to the viewer level on a per-project basis:
```

Comment:
```suggestion
Under Restrict access in Amplitude Data, specify from the following options to restrict permissions to the viewer level on a per-project basis:
```
@cyhk Is this in Data settings? Will it be a new tab?

Reply:
Yes, this will be a new tab in Data settings.

Reply:
To clarify - the Permissions tab is a new tab in Data settings, Restrict access in Amplitude Data is the title of the bottom section on the page, which looks like this:
<img width="1034" alt="Screenshot 2024-10-18 at 16 50 34" src="https://github.com/user-attachments/assets/3fa11bc3-9cea-439c-9f40-79201688e1d0">